src/tests/integration.test.js

```diff
@@ -0,0 +1,87 @@
+import express from 'express';
+import pkg from 'pg';
+const { Client } = pkg;
+import format from 'pg-format'
+
+import { make_client } from '../client.js';
+import { handlerNextJS } from '../index.js';
+
+// Scripts
+import { initTestDatabase } from './_init-test-database.js';
+import { createColumnAndIndex } from '../scripts/create-index.js';
+
+/**
+ * Test the server with client
+ */
+
+describe('client', () => {
+
+
+  const tableName = 'test_table';
+  const initTestDatabaseParams = { tableName, rowCount: 100 };
+
+  const client = new Client()
+  client.connect()
+
+  let serverListener;
+
+  beforeAll(async () => {
+    // Start the server
+
+    /**
+    * Start an express server with handlerNextJS on the route /api/search
+    */
+
+    const app = express();
+    const port = 3000;
+    app.use(express.json());
+
+    app.post('/api/search', handlerNextJS);
```

Comment:
How nice that this works in plain Express, outside NextJS. :smile: We should probably rename `handlerNextJS` to something more generic.

Reply:
Yeah :) It did take a little while of debugging. Because `express.json()` 

1. only worked when content type was set to json 
2. set body is type object instead of string

It's a good learning curve here :)

Reply:
I've renamed to searchHandler